### ISSUE TITLE
RTP not working

### ISSUE THREAD

**Issue by DorkyAF** [Cobblemon-Islands/Bugs-And-Issues#90]
Feature: /rtp 

Expected Behaviour: Teleports player to a random location.

Current Behaviour: RTP creeper sound effects play, but teleport not working [as of 20/01]

Video Recordings/Screenshots:

https://github.com/Cobblemon-Islands/Bugs-And-Issues/assets/156593734/1f8606d0-4c63-47ce-86b5-08e8271b4f51

**Comment by Adainish** [comment]
Is this still an active issue?

**Comment by Prawnsamdi** [comment]
I've not heard any complaints about it not working recently.
Not sure if fixed, however I do know removing cosmetics allowed players to rtp, so if cosmetics have been worked on in some way, this issue might be resolved

**Comment by Cathaldhus** [comment]
Yeah we came to the conclusion that the wings slot cosmetic was probably the issue. But unsure if it has been fixed, we've just been telling people to remove their cosmetic slot for the time being.

**Comment by Furqit** [comment]
It was fixed to my understanding? sysdm did add a function that removes the backpack automatically (and brings it back) when using rtp.

**Comment by Cathaldhus** [comment]
> It was fixed to my understanding? sysdm did add a function that removes the backpack automatically (and brings it back) when using rtp.

I haven't had any issues thus far since but someone yesterday brought up rtp wouldn't work, but I used it myself personally today and was fine. So I'm assuming it's fixed.
